From 20247942115b574d228462aeab6f6998d689bdba Mon Sep 17 00:00:00 2001
From: Eric Anholt <eric@anholt.net>
Date: Sat, 19 Mar 2016 23:13:07 -0700
Subject: [PATCH 088/101] arm64: Add platform selection for BCM2835
Git-commit: 628d30d1ccb897fee54a6f7312561cf2f6f72f09
Patch-mainline: v4.8-rc1
X-notes: Upstream commit has more selects
References: FATE#319481

Like ARCH_BCM_IPROC, we replicate the config option under arm64, but
with a much simpler set of selects under it.  Note that an ARCH_BCM
was also required, to get pinctrl/bcm/ to build.

Signed-off-by: Eric Anholt <eric@anholt.net>
Signed-off-by: Alexander Graf <agraf@suse.de>

Conflicts: arch/arm64/Kconfig.platforms

---
 arch/arm64/Kconfig.platforms | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index 4f4b164..776c159 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -1,5 +1,16 @@
 menu "Platform selection"
 
+config ARCH_BCM
+	bool
+
+config ARCH_BCM2835
+	bool "Broadcom BCM2835 family"
+	select ARCH_BCM
+	select PINCTRL_BCM2835
+	help
+	  This enables support for the Broadcom BCM2837 SoC.
+	  This SoC is used in the Raspberry Pi 3 device.
+
 config ARCH_BCM_IPROC
 	bool "Broadcom iProc SoC Family"
 	help
-- 
1.8.5.6

